packages/patternfly-4/content/design-guidelines/usage-and-behavior/toolbar/design.md

```diff
@@ -2,35 +2,115 @@
 path: "/design-guidelines/usage-and-behavior/toolbar"
 ---
 # Toolbar
-The toolbar contains multiple components that allow a user to manage and manipulate a data set. Data can be presented in a variety of views (table, list, card, etc.). PatternFly supports multiple toolbar variations that address a variety of use cases.
+Toolbars can be included to allow a user to manage and manipulate a data set. Data can be presented in any valid presentation, a table, a list, or a data visualization (chart), for example . The PatternFly Toolbar component is a flexible layout system that accommodates a variety of configurations that can be adapted to your specific needs.
```

Comment:
@mcarrano I think there's an extra space between sentence two and three, between "example" and "The".

Reply:
Fixed